script/seed.js

```diff
@@ -0,0 +1,42 @@
+const User = require('../server/models/User')
+const Post = require('../server/models/Post')
+const db = require('../server/db/db')
+const mongoose = require('mongoose')
+
+async function seed() {
```

Comment:
One thing you might consider is adding more `console.log()`s in the seed function to say what has been done at each step. This will remind you what the seed script does when you run it, and it will also help you troubleshoot if something breaks somehow.